feature-libs/customer-ticketing/core/connectors/converters.ts

```diff
@@ -1,7 +1,11 @@
 import { InjectionToken } from '@angular/core';
 import { Converter } from '@spartacus/core';
-import { TicketDetails } from '@spartacus/customer-ticketing/root';
+import { TicketDetails, TicketEvent } from '@spartacus/customer-ticketing/root';
 
 export const CUSTOMER_TICKETING_NORMALIZER = new InjectionToken<
   Converter<any, TicketDetails>
 >('CustomerTicketingNormalizer');
+
+export const CUSTOMER_TICKETING_SERIALIZER = new InjectionToken<
```

Comment:
Do you really need to set a serializer token? 

it's fine to have, but what is your logic for including it?
Are you going to mutate any of the ticketevent data to adhere to something that the backend accepts as an event?